client/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>hangoutsidesf</title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script src="dist/bundle.js"></script>
+  </body>
+</html>
```

Comment:
Missing new line at end of file

Reply:
Thanks for noticing this missing newline as well.